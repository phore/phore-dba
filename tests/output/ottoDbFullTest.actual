Test\Pupil::__set_state(/* #fefa */ [
	'id' => '12',
	'name' => 'peter',
	'surname' => 'dud\'e',
	'grade' => '4a',
	'birth' => '2000',
])
