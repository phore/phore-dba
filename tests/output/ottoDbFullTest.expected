Test\Pupil::__set_state(/* #b44f */ [
	'id' => '12',
	'name' => 'peter',
	'surname' => 'dud\'e',
	'grade' => '4a',
	'birth' => 2000,
])
